pkg/deployments/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Deployed `AuthorizerAdaptorEntrypoint` to all networks.
 - Deployed `AaveLinearPoolFactory` to all networks.
 - Deployed `PoolRecoveryHelper` to all networks.
+- Deployed `TimelockAuthorizer` to all networks.
```

Comment:
Should this say "to mainnet and goerli" then?

Reply:
Yep, that's right.

```suggestion
- Deployed `TimelockAuthorizer` to ethereum mainnet and goerli.
```

(Testing if suggestions work when you're not OP)